### DISCUSSION TITLE
Release of native binary sources

### DISCUSSION THREAD

**Discussion by oSumAtrIX** [Mahmud0808/Iconify#455] [Q&A] [answered]
Iconify ships native binaries, specifically AAPT2. Since the project is licensed under GPLv3, the source of these binaries should be public. AAPT2 does not compile to Android by default, so there must be a custom build process involved. The reason I am asking specifically is because I am also interested in using AAPT2 on Android under a GPLv3 license as well.

**Comment by Mahmud0808** [comment, marked as answer]
https://github.com/AAGaming00/aapt2

**Reply by oSumAtrIX** [reply]
That's a funny coincidence, considering I am coming from ReVanced, which the repository is forking from. In regards to that, AAPT2 is broken on ARMv7 as far as I know, which is something you may want to consider.

**Reply by Mahmud0808** [reply]
> AAPT2 is broken on ARMv7 as far as I know

Didn't know that, thanks for the heads-up.
